test/extensions/filters/http/rbac/rbac_filter_integration_test.cc

```diff
@@ -70,7 +70,7 @@ TEST_P(RBACIntegrationTest, RouteOverride) {
   config_helper_.addConfigModifier(
       [](envoy::config::filter::network::http_connection_manager::v2::HttpConnectionManager& cfg) {
         ProtobufWkt::Struct pfc;
-        ASSERT_TRUE(Protobuf::util::JsonStringToMessage(R"EOF({"disabled": true})EOF", &pfc).ok());
+        ASSERT_TRUE(Protobuf::util::JsonStringToMessage(R"EOF({})EOF", &pfc).ok());
```

Comment:
can remove the raw string markers: "{}"

Reply:
seems {} is needed(tried to remove it then test timeout..)

Reply:
sorry i meant replace `R"EOF({})EOF"` with just `"{}"`

Reply:
done.